test/parser.test.js

```diff
@@ -12,6 +15,34 @@ test('xlsxParser works with XLSX files', async t => {
   t.deepEqual(rows[0], ['number', 'string', 'boolean'])
 })
 
+test('csvParser iso8859 file encoding', async t => {
+  const path_ = 'test/fixtures/encodings/iso8859.csv'
+  const file = await File.load(path_)
+  const rows = await toArray(await csvParser(file))
+  t.deepEqual(rows[1], ['Réunion','ECS','1989','838462813'])
+})
+
+test('csvParser western-macos-roman file encoding', async t => {
+  const path_ = 'test/fixtures/encodings/western-macos-roman.csv'
+  const file = await File.load(path_)
+  const rows = await toArray(await csvParser(file))
+  t.deepEqual(rows[1], ['RŽunion','ECS','1989','838462813'])
```

Comment:
But this is wrong decoding as the first value of the row must be `Réunion`

Reply:
fixed tests